packages/cbioportal-ts-api-client/src/generated/CBioPortalAPIInternal.ts

```diff
@@ -102,6 +102,24 @@ export type ClinicalAttributeCountFilter = {
 
         'sampleListId': string
 
+};
+export type ClinicalData = {
```

Comment:
this is odd because we already have a ClinicalData type from the public client.  maybe this is true for other types that are used across clients.  not sure.

Reply:
This code is autogenerated. I think that the public and internal API docs are strictly separated by Swagger.